hometask-2/get_middle_charachter.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+def get_middle(str)
```

Comment:
actually, your method is not `getting` the middle, but prints it, so how about making this method actually `return` the middle, and at the end of your loop just printing the result?